onediff_comfy_nodes/modules/onediff_controlnet.py

```diff
@@ -77,7 +77,22 @@ def pre_run(self, model, percent_to_timestep_function):
             controlnet_config["hint_channels"] = self.control_weights[
                 "input_hint_block.0.weight"
             ].shape[1]
-            controlnet_config["operations"] = ControlLoraOps()
+
+            self.manual_cast_dtype = model.manual_cast_dtype
+            dtype = model.get_dtype()
+            if self.manual_cast_dtype is None:
+
+                class control_lora_ops(ControlLoraOps, comfy.ops.disable_weight_init):
+                    pass
+
+            else:
+
+                class control_lora_ops(ControlLoraOps, comfy.ops.manual_cast):
+                    pass
```

Comment:
这里的空行是为了强调保留的吗，是不是可以改成这样，更好看点：

```suggestion
            class lora_ops_disable_weight_init(ControlLoraOps, comfy.ops.disable_weight_init):
                pass
            class lora_ops_manual_cast(ControlLoraOps, comfy.ops.manual_cast):
                pass
            if self.manual_cast_dtype is None:
                control_lora_ops = lora_ops_disable_weight_init
            else:
                control_lora_ops = lora_ops_manual_cast
```

Reply:
###  `-`
self.manual_cast_dtype is None 为空 在 `85`行 comfy.ops.disable_weight_init  这个不会调用。
### `+`
使用在 `83` 行 ，self.manual_cast_dtype is None 为空 仍会调用 comfy.ops.disable_weight_init 从而会报错不存在 disable_weight_init 错误。